jdaviz/configs/default/plugins/export/export.py

```diff
@@ -52,6 +54,11 @@ class Export(PluginTemplateMixin, ViewerSelectMixin, SubsetSelectMixin,
     dev_plot_support = Bool(False).tag(sync=True)  # when enabling: add entries to docstring
     dev_multi_support = Bool(False).tag(sync=True)  # when enabling: add entries to docstring
 
+    dataset_items = List().tag(sync=True)
+    dataset_selected = Any().tag(sync=True)
+
+   # multiselect = Bool(False).tag(sync=True)
+

```

Comment:
this already inherits from `DatasetMultiSelectMixin` so should not need to add anything new